src/Img.tsx

```diff
@@ -46,7 +48,10 @@ export default function Img({
   // console.log({src, isLoading, resolvedSrc, useSuspense})
 
   // show img if loaded
-  if (src) return container(<img src={src} {...imgProps} />)
+  if (src)
+    return container(
+      <img referrerPolicy={referrerPolicy} src={src} {...imgProps} />
```

Comment:
We should not do anything if there is no referrerPolicy defined

Reply:
Undefined is good